### ISSUE TITLE
Query was not uploaded (crash)

### ISSUE THREAD

**Issue by Mismatchia** [Decimation/SmartImage#43]
SmartImage crashes when I run with any image. Here's the trace:
```
Unhandled exception. System.ArgumentException: Query was not uploaded (Parameter 'query')
   at SmartImage.Lib.SearchClient.GetSearchTasks(SearchQuery query, CancellationToken token)
   at SmartImage.Lib.SearchClient.RunSearchAsync(SearchQuery query, CancellationToken token, IProgress`1 p)
   at SmartImage.Mode.Shell.ShellMode.RunSearchAsync()
   at SmartImage.Mode.Shell.ShellMode.RunMain()
   at SmartImage.Mode.Shell.ShellMode.Run_Clicked()
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object)
   at Terminal.Gui.Application.MainLoopSyncContext.<>c__DisplayClass3_0.<Post>b__0()
   at Terminal.Gui.MainLoop.RunIdle()
   at Terminal.Gui.MainLoop.MainIteration()
   at Terminal.Gui.Application.RunMainLoopIteration(RunState&, Boolean, Boolean&)
   at Terminal.Gui.Application.RunLoop(RunState, Boolean )
   at Terminal.Gui.Application.Run(Toplevel, Func`2 )
   at Terminal.Gui.Application.Run(Func`2 )
   at SmartImage.Mode.Shell.ShellMode.RunAsync(Object )
   at SmartImage.Program.Main(String[])
   at SmartImage.Program.<Main>(String[])
```

I downloaded the latest release binary (v3.0.2 Release), and executed via windows powershell on windows 11. I copied an image to clipboard, and SmartImage pulled the correct file path into the Input field, upon clicking "run" SmartImage displays "uploading", draws the header for the results table, and then exits with the above trace. Config is default, aside from enabling the context menu.

Running SmartImage via the context menu opens, displays the same as the above, and exits (presumably with the same trace, however it closes before I can see).

**Comment by Decimation** [comment]
Can you try [v3.0.3-RC](https://github.com/Decimation/SmartImage/releases/tag/v3.0.3-rc)?

**Comment by Mismatchia** [comment]
Apologies for the late response, the issue is resolved in v3.0.3-RC.
